README.md

```diff
@@ -118,90 +112,40 @@ cd karmada
 
 #### 3. Deploy and run karmada control plane:
 
-run the following script: (It will create a host cluster by kind)
+run the following script:
 
 ```
-hack/local-up-karmada.sh
+# hack/local-up-karmada.sh
```

Comment:
Still one `#` here.

Reply:
this script needs root privilege for installing `kind` and `kubectl`

Reply:
I don't where requires the `root privilege`, let's figure it out and try to fix it.
I'll open an issue to track it.

Reply:
Done. https://github.com/karmada-io/karmada/issues/748